Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -44,6 +57,14 @@ public partial class MainAddActionsNavigationPage : Page
         APINavPage mAPINavPage = null;
         private bool applicationModelView;
 
+        /// <summary>
+        /// To keep the Panel Pages in sync with the panel
+        /// Ignore updating them when Panel is Collapsed.
+        /// Update them only when Panel is Expanded.
+        /// Auto Update When they are visible and Panel is expanded as well.
+        /// </summary>
+        public static bool isPanelExpanded = false;
```

Comment:
why this is needed if we check on each page if it visible? it can be that page will be Visible=True and Main page is collapsed?

Reply:
Yes. It happens that way and property IsVisible = true for the page, which is loaded in the Panel's frame

Reply:
ok, so keep the flag on main page for now